Cargo.toml

```diff
@@ -6,7 +6,7 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-rrplug = { git = "https://github.com/R2NorthstarTools/rrplug" }
+rrplug = "=3.0.0"
```

Comment:
Is there a reason to do `"=3.0.0"` vs just `"3.0.0"`? ^^"

Reply:
I don't quite follow semver so I might break smth accidentally in the future :>

Reply:
I thought `3.0.0` already means `=3.0.0`? ^^"

Reply:
iirc it means `3.0.*`

Reply:
Huh, TIL. Apparently it's even 

```
3.0.0  :=  >=3.0.0, <4.0.0
```

https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html


So yeah, in that case pinning is totally fine as rrplug version follows Northstar plugins version with major version number meaning minor changes could break compat within rrplug.